maven-scm-test/src/main/java/org/apache/maven/scm/ScmTestCase.java

```diff
@@ -60,7 +63,8 @@
 
     private SecDispatcher secDispatcher;
 
-    protected void setUp()
+    @Override
+    public void setUp()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@Before` (in JUnit 4 style)?

Reply:
fixed